modules/ethereumNode.js

```diff
@@ -218,12 +218,12 @@ class EthereumNode extends EventEmitter {
                     resolve();
                 }); 
             })
-                .then(() => {
-                    this.state = STATES.STOPPED;
-                    this._stopPromise = null;
-                });
+            .then(() => {
+                this.state = STATES.STOPPED;
+                this._stopPromise = null;
+            });
         } else {
-            log.debug('Disconnection already in progress, returning Promise.');
```

Comment:
Why has this been renamed? It can be re-connected after disconnection, hence why it is called that.